### PR TITLE
test: Add docker integration for --sysctl ip forward

### LINKED ISSUE #1491: Add docker integration for --sysctl ip forward
Add docker test for setting namespaced kernel parameters to turn on IP forwarding in the containers network namespace

### PR DESCRIPTION
**Pull request by GabyCT**
This adds a docker integration test for setting namespaced kernel
parameters to turn on IP forwarding in the containers network
namespace.

Fixes #1491

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/docker/sysctls_test.go [hunk 1]
```diff
@@ -54,4 +54,14 @@ var _ = Describe("sysctls", func() {
...
 		})
 	})
+
+	Context("sysctl for IP forwarding", func() {
+		It("should be applied", func() {
+			ipforwardValue := "1"
+			args = []string{"--name", id, "--rm", "--sysctl", "net.ipv4.ip_forward=" + ipforwardValue, Image, "cat", "/proc/sys/net/ipv4/ip_forward"}
+			stdout, _, exitCode = dockerRun(args...)
+			Expect(exitCode).To(Equal(0))
+			Expect(stdout).To(ContainSubstring(ipforwardValue))
```
Comment (devimc) [inline integration/docker/sysctls_test.go]: `Expect(strings.Trim(stdout, " \n\t")).To(ipforwardValue)`

**Comment by GabyCT** [comment]
@devimc changes applied

File: integration/docker/sysctls_test.go [hunk 2]
```diff
@@ -54,4 +56,14 @@ var _ = Describe("sysctls", func() {
...
 		})
 	})
+
+	Context("sysctl for IP forwarding", func() {
+		It("should be applied", func() {
+			ipforwardValue := "1"
+			args = []string{"--name", id, "--rm", "--sysctl", "net.ipv4.ip_forward=" + ipforwardValue, Image, "cat", "/proc/sys/net/ipv4/ip_forward"}
+			stdout, _, exitCode = dockerRun(args...)
+			Expect(exitCode).To(Equal(0))
+			Expect(strings.Trim(stdout, " \n\t")).To(ContainSubstring(ipforwardValue))
```
Comment (devimc) [inline integration/docker/sysctls_test.go]: `To(Equal(ipforwardValue))`

**Comment by GabyCT** [comment]
@devimc changes applied

**Pull request merged**